browser/ipfs/ipfs_tab_helper.cc

```diff
@@ -173,6 +236,48 @@ void IPFSTabHelper::LoadUrl(const GURL& gurl) {
   web_contents()->OpenURL(params);
 }
 
+#if !BUILDFLAG(IS_ANDROID)
+void IPFSTabHelper::LoadUrlForAutoRedirect(const GURL& gurl) {
+  auto set_user_data = [&](content::NavigationHandle* handle) {
```

Comment:
seems like this is used only once, so `handle->SetUserData` can just be put there.

Reply:
done